openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -495,4 +2981,4 @@ spec:
   configMap: serverless-operator
   displayName: Serverless Operator
   publisher: Red Hat
-  sourceType: internal
\ No newline at end of file
+  sourceType: internal
```

Comment:
I think you can cut this file down to just contain the 1.5.0 manifest.

Reply:
Yes, you are correct. But we can copy this file from serverless-operator and update by `openshift/olm/generate.sh`.
I tried to cut old version in the file by a script, but it will be complicated. So, we can put the full manifest here?